Modules/MFT/src/QcMFTTrackCheck.cxx

```diff
@@ -58,16 +58,16 @@ void QcMFTTrackCheck::beautify(std::shared_ptr<MonitorObject> mo, Quality checkR
     auto* histogram = dynamic_cast<TH1F*>(mo->getObject());
 
     if (checkResult == Quality::Good) {
```

Comment:
perhaps this whole block of code can be removed ?

Reply:
I removed it in the new commit